resources/assets/js/App.vue

```diff
@@ -83,13 +84,15 @@
     import {interceptors, post} from "./utilities/api";
     import SMSModal from './components/CustomSMSButton/SMSModal';
     import ChangeCustomerManagerModal from './components/modals/ChangeCustomerManagerModal';
+    import AutoLogout from './views/auth/AutoLogout';
 
     export default {
         components: {
             SideNav,
             Loader,
             SMSModal,
-            ChangeCustomerManagerModal
+            ChangeCustomerManagerModal,
```

Comment:
Why do we need a component for this? The app is slow enough, just log them out after no activities